backend/src/tasks/LookingGlass.ts

```diff
@@ -0,0 +1,286 @@
+import {
```

Comment:
For consistency with the other scans, let's rename this file to `lookingGlass.ts`